db/schema.rb

```diff
@@ -38,8 +38,8 @@
   end
 
   create_table "api_users", force: :cascade do |t|
-    t.string "name"
-    t.string "token"
+    t.string "name", default: "", null: false
+    t.string "token", default: "", null: false
```

Comment:
Where did this change come from?

Reply:
That's a good question, as I don't remember doing this, and it's certainly not part of this story. Am assuming a bad merge somewhere down the line - I will revert it, thanks